examples/node-echo/README.md

```diff
@@ -0,0 +1,54 @@
+Node.js `echo` Service on OpenShift!
+-----------------
+
+This example will serve an http response of various "machine" stats from the "machine" the Node.js app is running on to [http://host:8080](http://host:8080).
+
+### OpenShift setup ###
+
+I just used the docker all-in-one launch as described in the [origins project](https://github.com/openshift/origins).
+
+### The project ###
+
+If you don't have a project setup all ready, go ahead and take care of that
+
+        $ openshift ex new-project node-echo --display-name="node-echo" --description="Sample Node.js app" --admin=username
+
+That's it, project has been created.  Though it would probably be good to set your current project to this, such as:
+
+        $ osc project node-echo
+
+### The app ###
+
+Now let's pull in the app source code from [GitHub repo](https://github.com/sspeiche/node-echo) (fork if you like)
```

Comment:
Hrm - probably want the name to parallel the existing Ruby sample - I think it ends with -example or -sample

> On Apr 23, 2015, at 8:42 PM, Steve Speicher <notifications@github.com> wrote:
> 
> In examples/node-echo/README.md:
> 
> > +
> > +I just used the docker all-in-one launch as described in the [origins project](https://github.com/openshift/origins).
> > +
> > +### The project ###
> > +
> > +If you don't have a project setup all ready, go ahead and take care of that
> > +
> > +        $ openshift ex new-project node-echo --display-name="node-echo" --description="Sample Node.js app" --admin=username
> > +
> > +That's it, project has been created.  Though it would probably be good to set your current project to this, such as:
> > +
> > +        $ osc project node-echo
> > +
> > +### The app ###
> > +
> > +Now let's pull in the app source code from [GitHub repo](https://github.com/sspeiche/node-echo) (fork if you like)
> sure thing, would this be at github.com/openshift/node-echo or current folder where this readme is? I'd have to verify it would import/new-app ok
> 
> —
> Reply to this email directly or view it on GitHub.
>